Header header

geometry_msgs/Vector3 p
geometry_msgs/Vector3 v
geometry_msgs/Quaternion q
geometry_msgs/Vector3 eul_deg
geometry_msgs/Vector3 b_a
geometry_msgs/Vector3 b_g_deg
float32 b_alt
geometry_msgs/Vector3 l_b_r
geometry_msgs/Quaternion q_b_r
geometry_msgs/Vector3 eul_b_r_deg

